ui/apps/platform/src/Containers/Collections/CollectionsFormPage.tsx

```diff
@@ -212,7 +227,7 @@ function CollectionsFormPage({
             </Bullseye>
         );
     } else if (data) {
-        const pageTitle = pageAction.type === 'create' ? 'Create collection' : data.collection.name;
+        const pageName = pageAction.type === 'create' ? 'Create collection' : data.collection.name;
```

Comment:
Updated this constant name since `pageTitle` isn't really correct after this PR.